src/OpenTelemetry.Abstractions/Trace/Sampler/Samplers.cs

```diff
@@ -30,5 +30,15 @@ public sealed class Samplers
         /// Gets the sampler than never samples.
         /// </summary>
         public static ISampler NeverSample { get; } = new Internal.NeverSampleSampler();
+
+        /// <summary>
+        /// Gets the always on sampling decision.
+        /// </summary>
+        public static Decision AlwaysOnDecision { get; } = new Decision(true);
```

Comment:
nit: this is a single decision, it's not 'AlwaysOn', maybe we can do 'SampledIn'? Or since it's a struct, it't cheap to always do `new Decision(true)` without having static preconfigured value

Reply:
at least this shouldn't be public API - anyone can create a decision themselves